README.md

```diff
@@ -60,19 +92,19 @@ If the user answers "Allow once", then calling `RequestPermission` again during
 
 Once permission is requested, you can use the various methods and extension methods of IGeolocatorService.
 
-## Features
-
 ### Request the user's location
 
 Once you have obtained the location permission, it's a simple matter to obtain the location:
 
-```
-var location = await locationService.GetLocation(ct);
+``` cs
+if(isGranted){
```

Comment:
I add it because of the text below. New implementation shouldn't include it since it's recommanded ? I'll try to refactor it a little, to be more clear.